contributing.md

```diff
@@ -1,14 +1,51 @@
-#### How to reach us
-The mailing list for developers : systemroles@lists.fedorahosted.org
 
-[Subscribe to the mailing list here](https://lists.fedorahosted.org/admin/lists/systemroles.lists.fedorahosted.org/)
+# Contributing to Network @ Linux System Roles
 
-[Archive of the mailing list](https://lists.fedorahosted.org/archives/list/systemroles@lists.fedorahosted.org/)
+## Where to start
+- **Bugs and needed implementations** are listed on [Github
+  Issues](https://github.com/linux-system-roles/network/issus). Issues labeled
+with **help wanted** are likely to be suitable for new contributors!
```

Comment:
Please link `help wanted` to the query:
https://github.com/linux-system-roles/network/issues?q=is%3Aissue+is%3Aopen+label%3A%22help+wanted%22

Reply:
Done, thanks!